### PR TITLE
Use custom queue for RealmCollection observing

### PR DESCRIPTION
**Pull request by IgorRosocha**
Lately, we have experienced several crashes after updating `Realm` dependency in project which also uses `ACKReactiveExtensions`. Given PR should resolve the issue.

### CODE REVIEW

File: ACKReactiveExtensions/Realm/RealmExtensions.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ public extension Reactive where Base: RealmCollection {
             }
 
             func observe() -> NotificationToken? {
-                return self.base.observe(on: nil) { changes in
+                return self.base.observe(on: DispatchQueue(label: "RealmCollectionObserve")) { changes in
```
Comment (olejnjak) [inline ACKReactiveExtensions/Realm/RealmExtensions.swift]: ❓ I am not sure we want to create new queue every time this function is called, on the other hand I don't see any reason why not. Just point for discussion, I don't think it is an issue.

**Review by olejnjak — APPROVED** [review]
Nice job, must have been hell to be found 👍

**Review by janmisar — APPROVED** [review]

File: ACKReactiveExtensions/Realm/RealmExtensions.swift (hunk 1 above)
Comment (LukasHromadnik) [reply ACKReactiveExtensions/Realm/RealmExtensions.swift]: That's a great question. 👍  I don't think that it's gonna be an issue but I would try it anyway and see the result.

Comment (IgorRosocha) [reply ACKReactiveExtensions/Realm/RealmExtensions.swift]: Good point, I have changed the implementation to create only a single queue dedicated for `RealmCollection` observing.

**Review by olejnjak — APPROVED** [review]

**Pull request merged**